gym_car_race/SelfDriveEnv.py

```diff
@@ -533,7 +533,7 @@ def step(self, action):
         return observations, reward, self.done, {}
 
     def reset(self):
-        self.pos = self.config["car"]['position']
+        self.pos = [100, 100]
```

Comment:
Same logic as above

Reply:
Actually the problem with this is that with a `None`, the `_center_sensors()` wouldn't work because it needs an x and y value.

Reply:
`_center_sensors()` only gets called after the track is initialized meaning the car has inherited a valid initial position by then.

Reply:
The new stuff look good

> `_center_sensors()` only gets called after the track is initialized (with Qi yin's new changes) meaning the car has inherited a valid initial position by then.